test/e2e/features/basic.feature

```diff
@@ -61,6 +61,9 @@ Feature: Basic test
         Then stdout should match "(.*)[Ss]topped the instance"
         And executing "oc whoami" fails
         And kubeconfig is cleaned up
+        # Given CRC Cluster is stopped, When crc status is executed, Then crc should report that cluster is stopped without throwing any error
+        When executing "crc status"
+        Then stdout should match "(.*)Stopped"
```

Comment:
This is basically what next step does "checking that CRC is stopped"

See 

```
test/extended/crc/cmd/cmd.go
138 func CheckCRCStatus(state string) error {
```

Reply:
Oh, you're right.

If this scenario was already covered in e2e tests.